samples/sample_hevc_fei_abr/include/pipeline_hevc_fei.h

```diff
@@ -154,6 +157,8 @@ class CFeiTranscodingPipeline
     mfxStatus FillInputFrameInfo(mfxFrameInfo& fi);
 
     DISALLOW_COPY_AND_ASSIGN(CFeiTranscodingPipeline);
+
+    bool m_error; // used by asynchronous encoders to signal an error
```

Comment:
shouldn't it be some sort of atomic?

Reply:
I think it would be excessive. Let's just add 'volatile'

Reply:
sure